homeassistant/components/homematic/entity.py

```diff
@@ -268,6 +277,33 @@ def _update_variables(self, now):
         if state_change:
             self.schedule_update_ha_state()
 
+    def _update_duty_cycle(self, now):
+        """Retrieve duty cycle."""
+        if "DutyCycle" in self._variables:
+            # e.g. RaspberryMatic already sets DutyCycle system variable
+            self._duty_cycle = None
+            return
+        bidcos_interfaces = self._homematic.listBidcosInterfaces(self._name)
+        if bidcos_interfaces is None:
+            return
+        _LOGGER.debug("bidcos_interfaces: %s", bidcos_interfaces)
+        if len(bidcos_interfaces) == 0:
+            return
+        if len(bidcos_interfaces) > 1:
+            _LOGGER.warning(
+                "Found more than one bidcos interface - cannot retrieve duty cycle."
+            )
+            return
+        bidcos_interface = bidcos_interfaces[0]
+        if "DUTY_CYCLE" not in bidcos_interface:
+            _LOGGER.warning("DUTY_CYCLE not found in %s.", bidcos_interface)
+            return
+        duty_cycle = bidcos_interface["DUTY_CYCLE"]
+        if self._duty_cycle != duty_cycle:
+            _LOGGER.info("Retrieved new DutyCycle: %d", duty_cycle)
```

Comment:
This would be a debug-message at best. Since the value will be updated anyways it doesn't even need to be logged in my opinion.